include/open62541/client.h

```diff
@@ -109,6 +109,7 @@ struct UA_ClientConfig {
                                   * any matching SecurityPolicy. */
 
     UA_Boolean noSession;   /* Only open a SecureChannel, but no Session */
+    UA_Boolean noToken;     /* Don't check for matching user identity token */
```

Comment:
Make this an internal flag.
When we manually do a getendpoints, return all of them.
when we want to connect ourselves, of course we only consider the relevant ones..